### PR TITLE
Unit tests to make sure fonts are working as expected.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
@jrgm or @6a68, mind reviewing?
- Upgrade to the newest version of connect-fonts.
- Split off respondsWith into its own file so that it can be shared.
- For each locale, check whether all of the fonts.css requests are being served.
- For each served fonts.css, check to make sure embedded fonts are fetchable.

### CODE REVIEW

File: lib/wsapi_client.js [hunk 1]
```diff
@@ -98,7 +98,8 @@ exports.get = function(cfg, path, context, getArgs, cb) {
     injectHeaders(context, headers);
     injectCookies(context, headers);
 
-    if (typeof getArgs === 'object')
+    // skip the query string if getArgs is null (null is an object)
+    if (typeof getArgs === 'object' && getArgs !== null)
```
Comment (jaredhirsch) [inline lib/wsapi_client.js:102]: you could do if (getArgs && typeof getArgs === 'object')? Slightly less verbose...

Comment (shane-tomlinson) [inline lib/wsapi_client.js:102]: I dig it.

File: tests/fonts-request-test.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+ *
+ * The basic flow:
+ * Get a list of all production resources for all locales
+ * Search through the list of resources for all /fonts.css
+ * Check to make sure all /fonts.css are fetchable.
+ * Peak into each fonts.css file to look for embedded url(...)s
+ * Fetch each embedded URL to make sure the font is served.
+ */
+
+// Vows can only handle up to 50 tests per batch. After 50, it just
```
Comment (jaredhirsch) [inline tests/fonts-request-test.js:33]: wow, that's depressing

**Comment by jaredhirsch** [comment]
r+ looks good, works well. merging in.

File: tests/fonts-request-test.js (hunk 2 above)
Comment (shane-tomlinson) [inline tests/fonts-request-test.js:33]: I searched the vows code and cannot find a hard limit, but when I added 100 tests to a batch, only the first 50 would run. It was janked.

**Pull request merged**